src/Symfony/Component/VarDumper/Resources/functions/dump.php

```diff
@@ -41,8 +47,8 @@ function dd(...$vars): void
             header('HTTP/1.1 500 Internal Server Error');
         }
 
-        foreach ($vars as $v) {
-            VarDumper::dump($v);
+        foreach ($vars as $key => $v) {
+            VarDumper::dump($v, is_numeric($key) ? null : $key);
         }
```

Comment:
wouldn't be better to dump argument index as well here?

```suggestion
        $i = 0;
        foreach ($vars as $key => $v) {
            VarDumper::dump($v, is_numeric($key) ? "$i" : "$key@$i");
            $i++;
        }
```

Reply:
When there is only one argument, I wouldn't dump the index as label. But when there are more, it might be nice to dump the index+1 even when it's numeric (+1 to start from 1, and the index should be the key, no need for $i)
(same for dump() of course)

Reply:
I was :+1: on this proposition, but:

```php
$argc = count($vars);
foreach ($vars as $key => $v) {
    $displayLabel = 1 < $argc || !is_numeric($key);
    $label = is_numeric($key) ? $key+1 : $key;

    VarDumper::dump($v, $displayLabel ? $label : null);
}
```

This seems a bit complicated logic for the gain, what do you think?

Reply:
Like this?
```php
if (isset($vars[0]) && 1 === count($vars)) {
    VarDumper::dump($vars[0]);
} else {
    foreach ($vars as $key => $v) {
        VarDumper::dump($v, is_numeric($key) ? null : $key);
    }
}
```

Reply:
Yes, thank you!

![image](https://user-images.githubusercontent.com/2144837/205945445-4b562a95-3446-4429-b186-434c1397a771.png)